### PR TITLE
Assessment Correction Config fixed

### PR DESCRIPTION
**Pull request by utk14**

### CODE REVIEW

File: ansible/roles/data-products-deploy/defaults/main.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ channel: dev-test
 druid_broker_host: "{{groups['raw-broker'][0]}}"
 druid_rollup_broker_host: "{{groups['raw-broker'][0]}}"
 hierarchySearchServiceUrl: "{{ proto }}://{{ domain_name }}/action/content"
+
```
Comment (sowmya-dixit) [inline ansible/roles/data-products-deploy/defaults/main.yml:28]: Please remove unnecessary changes

File: ansible/roles/data-products-deploy/templates/model-config.j2 [hunk 2]
```diff
@@ -43,7 +43,7 @@ config() {
...
+         echo '{"search":{"type":"azure","queries":[{"bucket":"'$bucket'","prefix":"unique/raw/","endDate":"'$endDate'","delta":0}]},"model":"org.sunbird.analytics.model.report.AssessmentCorrectionModel","modelParams":{"parallelization":200,"druidConfig":{"queryType":"groupBy","dataSource":"content-model-snapshot","intervals":"1901-01-01T00:00:00+00:00/2101-01-01T00:00:00+00:00","granularity":"all","aggregations":[{"name":"count","type":"count","fieldName":"count"}],"dimensions":[{"fieldName":"identifier","aliasName":"identifier"}],"filters":[{"type":"equals","dimension":"contentType","value":"SelfAssess"}],"descending":"false"},"fileOutputConfig":{"to":"file","params":{"file":"/mount/data/analytics/tmp/assessment-correction/skippedEvents"}},"sparkCassandraConnectionHost":"'$sunbirdPlatformCassandraHost'"},"output":[{"to":"kafka","params":{"brokerList":"'$brokerIngestionList'","topic":"'$assessTopic'"}}],"parallelization":200,"appName":"Assessment Correction Model"}'
```
Comment (sowmya-dixit) [inline ansible/roles/data-products-deploy/templates/model-config.j2:46]: Please use variable instead of hardcoding file path

File: ansible/roles/data-products-deploy/defaults/main.yml (hunk 1 above)
Comment (utk14) [reply ansible/roles/data-products-deploy/defaults/main.yml]: Done

File: ansible/roles/data-products-deploy/templates/model-config.j2 (hunk 2 above)
Comment (utk14) [reply ansible/roles/data-products-deploy/templates/model-config.j2]: Done

**Review by sowmya-dixit — DISMISSED** [review]

**Review by sowmya-dixit — APPROVED** [review]
